library/core/src/iter/sources.rs

```diff
@@ -5,9 +5,8 @@ use super::{FusedIterator, TrustedLen};
 
 /// An iterator that repeats an element endlessly.
 ///
-/// This `struct` is created by the [`repeat`] function. See its documentation for more.
-///
-/// [`repeat`]: fn.repeat.html
+/// This `struct` is created by [`repeat`].
+/// See its documentation for more.
```

Comment:
You should put this on a new line, or rustdoc will put it as part of the summary line.
```suggestion
///
/// See its documentation for more.
```

Reply:
The summary line is above this.

> *An iterator that repeats an element endlessly.*